training/src/anemoi/training/diagnostics/mlflow/utils.py

```diff
@@ -120,3 +130,124 @@ def expand(val: dict | list) -> dict[str, Any]:
         else:
             expanded_params[key] = expand(value)
     return expanded_params
+
+
+def log_hyperparams_as_mlflow_artifact(client: MlflowClient, run_id: str, params: dict[str, Any] | Namespace) -> None:
```

Comment:
Why did we have to move those methods to a utils script? Initially those were private methods, but if we were to need to use them outside of the logger could we have not kept them there and make them staticmethods?

Reply:
I put it back as static methods in the Logger class.